frontend/packages/console-shared/src/hooks/previous.ts

```diff
@@ -1,5 +1,5 @@
 import { useMemo } from 'react';
 
-export const usePrevious = <P = any>(value: P, deps: any[] = []): P =>
+export const usePrevious = <P = any>(value: P, deps: any[] = [value]): P =>
```

Comment:
Maybe I'm misunderstaning, but won't it always return the current value now instead of the previous?

Reply:
yes, this actually does not make sense. Sorry i guess its too late for me today :)